v2/googlecloud-to-googlecloud/docs/BigtableChangeStreamsToPubSub/README.md

```diff
@@ -0,0 +1,51 @@
+| modType         | STRING        | STRING           | STRING    | N         | Modification type: {SET_CELL, DELETE_CELLS, DELETE_FAMILY}. DeleteFromRow mutation is converted into a series of DELETE_FROM_FAMILY entries. |
+| isGC            | BOOL          | BOOLEAN          | BOOL      | Y         | TRUE indicates that mutation was made by garbage collection in CBT                                                                           |
+| tiebreaker*     | INT32         | INT              | INTEGER   | N         | CBT tie-breaker value. Used for conflict resolution if two mutations are committed at the sametime.                                          | 
+| commitTimestamp | TIMESTAMP     | TIMESTAMP-MILLIS | STRING    | N         | Time when CBT wrote this mutation to a tablet                                                                                                |
+| columnFamily    | STRING        | STRING           | STRING    | N         | CBT column family name                                                                                                                       |
+| column          | BYTES         | BYTES            | STRING    | Y         | CBT column qualifier                                                                                                                         |
+| timestamp       | INT64         | LONG             | STRING    | Y         | CBT cell’s timestamp in microseconds. Type is determined by _writeNumericTimestamps_ pipeline option                                         |
+| timestampFrom   | NT64          | LONG             | STRING    | Y         | Time range start in microseconds (inclusive) for a DeleteFromColumn mutation. Type is determined by _writeNumericTimestamps_ pipeline option |
+| timestampTo     | INT64         | LONG             | STRING    | Y         | Time range end in microseconds (exclusive) for a DeleteFromColumn mutation. Type is determined by _writeNumericTimestamps_ pipeline option   |                             
+| value           | BYTES         | BYTES            | STRING    | Y         | Bigtable cell value. Not specified for delete operations                                                                                     |
+| sourceInstance           | STRING        | STRING           | STRING    | N         | The ID of the Cloud Bigtable instance that contains the table.                                                                               |
+| sourceCluster           | STRING        | STRING           | STRING    | N         | The ID of the cloud Bigtable cluster where mutation was made.                                                                                |
+| sourceTable           | STRING        | STRING           | STRING    | N         | The Cloud Bigtable table to read the change streams from.                                                                                    |
+
```

Comment:
sourceInstance: The name of the Bigtable instance registered the mutation. 
sourceCluster: The name of the Bigtable cluster registered the mutation. 
sourceTable: The name of the Bigtable table registered the mutation.